README.md

```diff
@@ -58,3 +58,12 @@ To run magic, navigate to the directory where it is installed eg. `cd ~/EE431_To
 ### Step 2: Launch Magic
 Once in the container, simply type `magic` and the program should start up with a gui.
 
+# Verify installation
```

Comment:
Doesn't this verify that xschem is working, not magic? I just finished the config work for magic so I can add some verification stuff that determines if magic is running with or without the PDK.